### PR TITLE
Minor CLI improvements

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

I am planning more refactorings to the commands also in the context of the required changes for the resolved configuration, so there is more to come.

### CODE REVIEW

**Review by fviernau — APPROVED** [review]

File: cli/src/main/kotlin/OrtCommand.kt [hunk 1]
```diff
@@ -39,4 +42,16 @@ abstract class OrtCommand(name: String, help: String) : CliktCommand(name = name
     override val type = commandName
 
     protected val ortConfig by requireObject<OrtConfiguration>()
+
+    /**
+     * Validates that the provided [outputFiles] can be used. Throws a [UsageError] otherwise.
+     */
+    protected fun validateOutputFiles(outputFiles: Collection<File>) {
+        if (!ortConfig.forceOverwrite) {
```
Comment (sschuberth) [inline cli/src/main/kotlin/OrtCommand.kt:51]: Nit: Maybe return early in this case to reduce nesting?

File: cli/src/main/kotlin/OrtCommand.kt [hunk 2]
```diff
@@ -50,7 +51,11 @@ abstract class OrtCommand(name: String, help: String) : CliktCommand(name = name
         if (!ortConfig.forceOverwrite) {
             val existingOutputFiles = outputFiles.filter { it.exists() }
             if (existingOutputFiles.isNotEmpty()) {
-                throw UsageError("None of the output files $existingOutputFiles must exist yet.", statusCode = 2)
+                throw UsageError(
+                    text = "None of the output files $existingOutputFiles must exist yet. To overwrite output files " +
+                            "set the 'forceOverwrite' option in '$ORT_CONFIG_FILENAME'.",
```
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply cli/src/main/kotlin/OrtCommand.kt:56]: Do we actually still need the CLI option if you can also do `-P ort.forceOverwrite=true`?

[the remark this answers is not in the document]
Comment (sschuberth) [reply cli/src/main/kotlin/OrtCommand.kt:56]: Good point, probably not.

[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply cli/src/main/kotlin/OrtCommand.kt:56]: The option was already deprecated, see #6778.

**Pull request merged**